scripts/test-dssspell-forge.sh

```diff
@@ -18,16 +19,20 @@ done
 DSS_EXEC_LIB=$(< DssExecLib.address)
 echo "Using DssExecLib at: $DSS_EXEC_LIB"
 export FOUNDRY_LIBRARIES="lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:$DSS_EXEC_LIB"
-export FOUNDRY_OPTIMIZER=true
+export FOUNDRY_OPTIMIZER=false
```

Comment:
Noting here that this PR will set the optimizer to false. The 'true' status is a remnant from the 2023-07-12 and does not represent a change made by this PR.

OK

Reply:
I actually would suggest to enable it in the spell itself (as part of the cleanup) to not mess up with the git history

Reply:
I wasn't sure if I should make this change here or leave it for the spell PR itself. Technically, it's a cleanup because the last spell only added it to be able to deploy the code.

I already reverted this change though.